cmd/runtimetest/main.go

```diff
@@ -370,6 +371,52 @@ func validateMountsExist(spec *rspec.Spec) error {
 	return nil
 }
 
+func getActualCgroupPath() (string, error) {
+	f, err := os.Open("/proc/self/cgroup")
+	if err != nil {
+		return "", err
+	}
+	defer f.Close()
+
+	s := bufio.NewScanner(f)
+	for s.Scan() {
+		if err := s.Err(); err != nil {
+			return "", err
+		}
+
+		text := s.Text()
+		parts := strings.Split(text, ":")
+		return parts[2], nil
+	}
+	return "", fmt.Errorf("can not get cgroup path")
+}
+
+func validateCgroupsPath(spec *rspec.Spec) error {
+	logrus.Debugf("validating cgroupsPath")
+	expectedPath := spec.Linux.CgroupsPath
+	if expectedPath == nil {
+		return nil
+	}
+	*expectedPath = strings.Replace(*expectedPath, ":", "/", -1)
```

Comment:
The problem is that we're applying stronger limits on the cgroupsPath logic than actually exist in the spec. The spec doesn't say how relative paths must be handled -- the only constraint is that it join the same place in the hierarchy consistently. So we can test that if you want, but testing any case where you're checking the actual value of the cgroup paths against some predefined value and `!filepath.IsAbs` is going beyond the specification -- even if runC will pass the tests no other runtime is obligated to.